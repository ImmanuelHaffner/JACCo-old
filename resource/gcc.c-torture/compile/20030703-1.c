


int SAD_Block(int *);
void MBMotionEstimation(int *act_block, int block)
{
    SAD_Block(act_block + ( (8 * (block == 1 || block == 3))
                          + (8 * (block == 2 || block == 3))));
}
