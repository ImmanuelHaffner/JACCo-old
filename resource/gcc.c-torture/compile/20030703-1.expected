0
20030703-1.c:4:1 keyword int
20030703-1.c:4:5 identifier SAD_Block
20030703-1.c:4:14 punctuator (
20030703-1.c:4:15 keyword int
20030703-1.c:4:19 punctuator *
20030703-1.c:4:20 punctuator )
20030703-1.c:4:21 punctuator ;
20030703-1.c:5:1 keyword void
20030703-1.c:5:6 identifier MBMotionEstimation
20030703-1.c:5:24 punctuator (
20030703-1.c:5:25 keyword int
20030703-1.c:5:29 punctuator *
20030703-1.c:5:30 identifier act_block
20030703-1.c:5:39 punctuator ,
20030703-1.c:5:41 keyword int
20030703-1.c:5:45 identifier block
20030703-1.c:5:50 punctuator )
20030703-1.c:6:1 punctuator {
20030703-1.c:7:5 identifier SAD_Block
20030703-1.c:7:14 punctuator (
20030703-1.c:7:15 identifier act_block
20030703-1.c:7:25 punctuator +
20030703-1.c:7:27 punctuator (
20030703-1.c:7:29 punctuator (
20030703-1.c:7:30 constant 8
20030703-1.c:7:32 punctuator *
20030703-1.c:7:34 punctuator (
20030703-1.c:7:35 identifier block
20030703-1.c:7:41 punctuator ==
20030703-1.c:7:44 constant 1
20030703-1.c:7:46 punctuator ||
20030703-1.c:7:49 identifier block
20030703-1.c:7:55 punctuator ==
20030703-1.c:7:58 constant 3
20030703-1.c:7:59 punctuator )
20030703-1.c:7:60 punctuator )
20030703-1.c:8:27 punctuator +
20030703-1.c:8:29 punctuator (
20030703-1.c:8:30 constant 8
20030703-1.c:8:32 punctuator *
20030703-1.c:8:34 punctuator (
20030703-1.c:8:35 identifier block
20030703-1.c:8:41 punctuator ==
20030703-1.c:8:44 constant 2
20030703-1.c:8:46 punctuator ||
20030703-1.c:8:49 identifier block
20030703-1.c:8:55 punctuator ==
20030703-1.c:8:58 constant 3
20030703-1.c:8:59 punctuator )
20030703-1.c:8:60 punctuator )
20030703-1.c:8:61 punctuator )
20030703-1.c:8:62 punctuator )
20030703-1.c:8:63 punctuator ;
20030703-1.c:9:1 punctuator }
